### PR TITLE
coffeescript 1.11.1 (new formula)

### PR DESCRIPTION
**Pull request by cglong**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Aliases/coffeescript [hunk 1]
```diff
@@ -0,0 +1 @@
+../Formula/coffee-script.rb
```
Comment (bfontaine) [inline Aliases/coffeescript]: I think it should be named `coffeescript` directly since that’s the name upstream uses.

**Review by bfontaine — DISMISSED** [review]

File: Aliases/coffeescript (hunk 1 above)
Comment (cglong) [inline Aliases/coffeescript]: I'm tempted to agree with you @bfontaine; the reason I went with this naming scheme originally is that, while the `git` repo calls it [`coffeescript`](https://github.com/jashkenas/coffeescript), the main entry point for users (`npm`) calls it [`coffee-script`](https://www.npmjs.com/package/coffee-script). I'm fine with either name, so let me know your thoughts and I can make any changes if necessary 😄

Comment (MikeMcQuaid) [inline Aliases/coffeescript]: Let's go for `coffeescript`, thanks!

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks again for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**